pkg/roll/execute.go

```diff
@@ -147,16 +147,22 @@ func (m *Roll) Rollback(ctx context.Context) error {
 }
 
 // create view creates a view for the new version of the schema
-func (m *Roll) createView(ctx context.Context, version string, name string, table schema.Table) error {
+func (m *Roll) createView(ctx context.Context, version, name string, table schema.Table) error {
 	columns := make([]string, 0, len(table.Columns))
 	for k, v := range table.Columns {
 		columns = append(columns, fmt.Sprintf("%s AS %s", pq.QuoteIdentifier(v.Name), pq.QuoteIdentifier(k)))
 	}
 
+	withOptions := ""
```

Comment:
Could you add a comment here to explain security_invoker? 🙏

Reply:
done